pymeasure/instruments/tcpowerconversion/tccxn.py

```diff
@@ -0,0 +1,427 @@
+
+
+class PresetChannel(Channel):
+    values = values
+
+    load_capacity = Instrument.control(
+        "GU\x00{ch:c}\x00\x00", "TD{ch:c}\x01\x00%c",
+        """Control the percentage of full-scale value of the load capacity preset.""",
+        preprocess_reply=lambda d: struct.unpack(">H", d[2:4]),
+        validator=strict_discrete_set,
+        values=range(101),
+    )
+
+    tune_capacity = Instrument.control(
+        "GU\x00{ch:c}\x00\x00", "TD{ch:c}\x02\x00%c",
+        """Control the percentage of full-scale value of the tune capacity preset.""",
+        preprocess_reply=lambda d: struct.unpack(">H", d[4:6]),
+        validator=strict_discrete_set,
+        values=range(101),
+    )
+
+
+class CXN(Instrument):
+    """T&C Power Conversion AG Series Plasma Generator CXN
+    (also rebranded by AJA International Inc as 0113 GTC or 0313 GTC)
+
+    Connection to the device is made through an RS232 serial connection.
+    The communication settings are fixed in the device at 38400, stopbit one,
+    parity none. The device uses a command response system where every receipt
+    of a command is acknowledged by returning a '*'. A '?' is returned to
+    indicates the command was not recognized by the device.
+
+    A command messages always consists of the following bytes (B):
+    1B - header (always 'C'),
+    1B - address (ignored),
+    2B - command id,
+    2B - parameter 1,
+    2B - parameter,
+    2B - checksum
+
+    A response message always consists of:
+    1B - header (always 'R'),
+    1B - address of the device,
+    2B - length of the data package,
+    variable length data,
+    2B - checksum
+    response messages are received after the acknowledge byte.
+
+    :param adapter: pyvisa resource name of the instrument or adapter instance
+    :param string name: Name of the instrument.
+    :param kwargs: Any valid key-word argument for Instrument
+
+    .. Note::
+
+        In order to enable setting any parameters one has to request control
+        and periodically (at least once per 2s) poll any value from the device.
+        Failure to do so will mean loss of control and the device will reset
+        certain parameters (setpoint, disable RF, ...). If no value should be polled
+        but control should remain active one can also use the ping method.
+    """
+    # use predefined values method to allow reusing in the Channels
+    values = values
+    presets = Instrument.ChannelCreator(PresetChannel, (1, 2, 3, 4, 5, 6, 7, 8, 9),
+                                        prefix="preset_")
+
+    def __init__(self, adapter, name="T&C RF sputtering power supply", address=0, **kwargs):
+        self.address = address
+        super().__init__(adapter,
+                         name,
+                         includeSCPI=False,
+                         write_termination="",
+                         read_termination="",
+                         asrl=dict(baud_rate=38400),
```

Comment:
I see `baud_rate=38400` extremely simple, clear and working.
While I found `asrl=dict(baud_rate=38400)` a bit suspicious to the casual reader.
Why would I need to define a python dictionary and assign to `asrl` parameter when I can be deliciously direct and say `baud_rate=38400` ?

Reply:
We discussed it in conversation: https://github.com/pymeasure/pymeasure/pull/800#discussion_r1055470564

Reply:
For sure, the code proposed it works, but my personal taste is to avoid making code more complex to address extreme cases. In the specific case of ethernet to serial, I am for using a dedicated adapter, but this discussion can continued on #815.

Reply:
@bmoneke after discussion in #815 seems to be concluded: Can this now be kept as is? 
I guess with the outcome of #815 it could either stay as is or also use `kwargs.setdefault(baud_rate, 38400)` without  `asrl`. 
what is your preferred way?

Reply:
kwargs.setdefault will have the same effect as setting it directly. Type specific stuff (like baud_rate) should be in the type dict.

Reply:
Thanks for merging this and sorry to keep commenting on something closed, but if you do not mind I would like to understand what you mean by this:

> kwargs.setdefault will have the same effect as setting it directly. 

This is not the first time you say this, but it seems not to be true. If you use `setdefault` then one can overwrite the value. If I put it directly in the super call  and try to overwrite it will give a TypeError.

```
class testkwarg(dict):
    def __init__(self, **kwargs):
        kwargs.setdefault("a", 2)
        super().__init__(b=2, **kwargs)

testkwargs(a=3)  # works with result {'b': 2, 'a': 3}
testkwargs(b=3)  # raises TypeError
```

Reply:
I agree that you can override.

I meant, that you setting the baud rate via setdefault will prevent your Ethernet adapters from working, as you cannot "unset" the baud rate.

In this regard, both options are the same.